RetailCoder.VBE/Inspections/InspectionBase.cs

```diff
@@ -81,6 +83,28 @@ protected virtual IEnumerable<Declaration> BuiltInDeclarations
             get { return State.AllDeclarations.Where(declaration => declaration.IsBuiltIn); }
         }
 
+        protected bool HasIgnoreAnnotation(VBComponent component, int line)
```

Comment:
That is in Declaration.  However, it only checks whether a given declaration was disabled, while I need this for the parser tree inspections, because we aren't working off declarations.